charts/piraeus/templates/operator-satelliteset.yaml

```diff
@@ -20,6 +20,9 @@ spec:
   kernelModuleInjectionMode: {{ .Values.operator.satelliteSet.kernelModuleInjectionMode | quote }}
   kernelModuleInjectionImage: {{ .Values.operator.satelliteSet.kernelModuleInjectionImage | quote }}
   kernelModuleInjectionResources: {{ .Values.operator.satelliteSet.kernelModuleInjectionResources | toJson }}
+  {{- if typeIs "bool" .Values.operator.satelliteSet.kernelModuleInjectionMountUsrSrc }}
```

Comment:
Why the extra check here? We don't have it for other values including `dbUseClientCert`, which is also a boolean.

Reply:
See above. If I want to avoid a mandatory CRD upgrade, I have to leave the field out of the CR unless explicitly set. I also can't apply a simple empty check, as that fails when it was explicitly set to true/false (wrong type). I also can't just compare it to a string (type error again). I also can't check if it's false-y, as that would mean an explicit `false` would not be set :/